test/navigation/TrusteeOrganisationNavigatorSpec.scala

```diff
@@ -18,85 +18,201 @@ package navigation
+          .mustBe(AddressUkYesNoController.onPageLoad(index, fakeDraftId))
+      }
 
-    "Is address in UK page -> YES -> UK address page" in {
-      val answers = emptyUserAnswers
-        .set(AddressUkYesNoPage(index), true).success.value
+      "Do you know address page -> NO -> Check your answers page" in {
+        val answers = baseAnswers
+          .set(AddressYesNoPage(index), false).success.value
 
-      navigator.nextPage(AddressUkYesNoPage(index), fakeDraftId, answers)
-        .mustBe(UkAddressController.onPageLoad(index, fakeDraftId))
-    }
+        navigator.nextPage(AddressYesNoPage(index), fakeDraftId, answers)
+          .mustBe(CheckDetailsController.onPageLoad(index, draftId))
+      }
 
-    "Is address in UK page -> NO -> International address page" in {
-      val answers = emptyUserAnswers
-        .set(AddressUkYesNoPage(index), false).success.value
+      "Is address in UK page -> YES -> UK address page" in {
+        val answers = baseAnswers
+          .set(AddressUkYesNoPage(index), true).success.value
 
-      navigator.nextPage(AddressUkYesNoPage(index), fakeDraftId, answers)
-        .mustBe(InternationalAddressController.onPageLoad(index, fakeDraftId))
-    }
+        navigator.nextPage(AddressUkYesNoPage(index), fakeDraftId, answers)
+          .mustBe(UkAddressController.onPageLoad(index, fakeDraftId))
+      }
+
+      "Is address in UK page -> NO -> International address page" in {
+        val answers = baseAnswers
+          .set(AddressUkYesNoPage(index), false).success.value
+
+        navigator.nextPage(AddressUkYesNoPage(index), fakeDraftId, answers)
+          .mustBe(InternationalAddressController.onPageLoad(index, fakeDraftId))
+      }
+
+      "UK address page -> Check your answers page" in {
+        navigator.nextPage(UkAddressPage(index), fakeDraftId, baseAnswers)
+          .mustBe(CheckDetailsController.onPageLoad(index, draftId))
+      }
+
+      "International address page -> Check your answers page" in {
+        navigator.nextPage(InternationalAddressPage(index), fakeDraftId, baseAnswers)
+          .mustBe(CheckDetailsController.onPageLoad(index, draftId))
+      }
 
-    "UK address page -> Check your answers page" in {
-      navigator.nextPage(UkAddressPage(index), fakeDraftId, emptyUserAnswers)
-        .mustBe(CheckDetailsController.onPageLoad(index, draftId))
     }
+    "A 5mld Trust" must {
+
+      val baseAnswers = emptyUserAnswers.copy(is5mldEnabled = true)
+
+      "Country Of Residence Yes No page -> Yes -> Country Of Residence UK yes no page" in {
+        val answers = baseAnswers
+          .set(CountryOfResidenceYesNoPage(index), true).success.value
+
+        navigator.nextPage(CountryOfResidenceYesNoPage(index), fakeDraftId, answers)
+          .mustBe(mld5Rts.CountryOfResidenceInTheUkYesNoController.onPageLoad(index, fakeDraftId))
+      }
+
+      "Country Of Residence Yes No page -> No -> Check details page" in {
```

Comment:
Double check that this isn't supposed to go to `AddressYesNoController`. I think the site map may be misleading